RecursiveExtractor/FileEntry.cs

```diff
@@ -73,9 +77,22 @@ public FileEntry(string name, Stream inputStream, FileEntry? parent = null, bool
             }
             else
             {
-                // Back with a temporary filestream, this is optimized to be cached in memory when possible
-                // automatically by .NET
-                Content = new FileStream(Path.GetTempFileName(), FileMode.Create, FileAccess.ReadWrite, FileShare.ReadWrite, 4096, FileOptions.DeleteOnClose);
+                try
```

Comment:
FileStreams end up writing a lot of junk to disk despite the comment here it appears not to be true in testing or you may be extracting something potentially malicious you want to keep in memory.  The option has existed for a while it just wasn't being respected.

Reply:
You should now be able to make it use memory streams whenever possible (file less than intmax size) with int.MaxValue

Reply:
ok, my point was FileStreams are memory backed anyways so didnt make much sense, but if you are guarding against malicious stuff on temp disk files, it might make sense...(although only if you are checking if the stuff is malicious before writing it do disk anyways).

Reply:
It would be up the end user to be careful about malicious files, but if they are particularly concerned (for example you want to analyze them without your anti virus deleting them off disc) you have the option.